spec/unit/shoryuken/error_handler_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'spec_helper'
+require 'airbrake/shoryuken/error_handler'
+
+RSpec.describe Airbrake::Shoryuken::ErrorHandler do
+  let(:error) { AirbrakeTestError.new('shoryuken error') }
+  let(:body) { { message: 'message' } }
+  let(:notice) { double 'notice' }
+  let(:worker) { FooWorker.new }
+  let(:endpoint) do
+    'https://airbrake.io/api/v3/projects/113743/notices?key=fd04e13d806a90f96614ad8e529b2822'
+  end
+
+  class FooWorker; end
+
+  def wait_for_a_request_with_body(body)
+    wait_for(a_request(:post, endpoint).with(body: body)).to have_been_made.once
+  end
+
+  before do
+    stub_request(:post, endpoint).to_return(status: 201, body: '{}')
+  end
+
+  context "when there's an error" do
+    it 'notifies' do
+      expect do
+        subject.call(worker, nil, nil, body) { raise error }
+      end.to raise_error(error)
+
+      wait_for_a_request_with_body(/"message":"shoryuken\serror"/)
```

Comment:
> Nitpick: \s isn't needed.

@kyrylo are you suggesting to change `\s` with a whitespace? I copied over the sidekiq example.
 
> This will pass even for the batch test. 

I don't think it would, the line below checks for `body`, the specific test for batch checks for `batch`.